install_config/configuring_aws.adoc

```diff
@@ -310,18 +310,26 @@ There are four types of resources that need to be tagged:
 [[aws-tagging-an-existing-cluster]]
 === Tagging an Existing Cluster
 
-A cluster uses the value of the `kubernetes.io/cluster/<name>,Value=<clusterid>` tag to determine which
-resources belong to the cluster. Therefore, you must tag all resources with the
-key `kubernetes.io/cluster/<name>,Value=<clusterid>` and have the same value for that key.
+A cluster uses the value of the `kubernetes.io/cluster/<name>,Value=<clusterid>`
+tag to determine which resources belong to the AWS cluster. Therefore, tag all
+relevant resources with the `kubernetes.io/cluster/<name>,Value=<clusterid>`
+key, and have the same value for that key.
 
-. Tag all instances with `kubernetes.io/cluster/<name>,Value=<clusterid>` and a value to be used as the cluster ID.
-. Tag any security groups with `kubernetes.io/cluster/<name>,Value=<clusterid>` and the same value used for the instances.
-. Tag any load balancers with `kubernetes.io/cluster/<name>,Value=<clusterid>` and the same value used for the instances.
+. Tag all hosts with `kubernetes.io/cluster/<name>,Value=<clusterid>` and a value to be used as the cluster ID.
```

Comment:
If you add a note before these steps, I'd remove "and a value to be used as the cluster ID" and similar from these three steps. (I think you're saying that the clusterID value is constant in all three, but I'm not sure.)

Reply:
I see what you mean. I've updated the points to take away the fluff.